docs/git-compatibility.md

```diff
@@ -9,6 +9,53 @@ See `jj help git` for help about the `jj git` family of commands, and e.g.
 briefer help).
 
 
+## Supported features
+
+The following list describes which Git features Jujutsu is compatible with. For
+a comparison with Git, including how workflows are different, see the
+[Git-comparison doc](git-comparison.md).
+
+* Configuration: No, the only configuration from Git (e.g. in `~/.gitconfig`)
+  that's respected is the configuration of remotes. Feel free to file a bug
+  if you miss any particular configuration options.
+* Authentication: Partial. Only `ssh-agent` or a password-less key file at
+  `~/.ssh/id_rsa` (and only at exactly that path).
+* Branches: Yes. You can read more about
+  [how branches work in Jujutsu](branches.md)
+  and [how they interoperate with Git](#branches).
+* Tags: Partial. You can check out tagged commits by name (pointed to be either
+  annotated or lightweight tags), but you cannot create new tags.
+* .gitignore: Partial. No support for ignores in `.git/info/exclude`
+  ([#65](https://github.com/martinvonz/jj/issues/65)) or via `core.excludesfile`
+  config ([#87](https://github.com/martinvonz/jj/issues/87)). Also, it uses a
+  native implementation, so please report a bug if you notice any difference
+  compared to `git`.  
+* .gitattributes: No. There's [#53](https://github.com/martinvonz/jj/issues/53)
+  about adding support for at least the `eol` attribute.
+* Merge commits: Yes, including octopus merges (i.e. with more than 2 parents).
+* Detached HEAD: Yes. Jujutsu supports anonymous branches, so this is a natural
+  state.
+* Orphan branch: Yes. Jujutsu has a virtual root commit that appears as parent
+  of all commits Git would call "root commits".
+* Staging area. Kind of. The staging area will be ignored. For example,
```

Comment:
```suggestion
* Staging area: Kind of. The staging area will be ignored. For example,
```

Worth adding a link to https://github.com/martinvonz/jj/blob/main/docs/git-comparison.md#the-index in this section.

Reply:
Good point, will add the link.